src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+* **Future-Proofed API**: CAPI's open and vendor-neutral API ensures our platform's viability in the long term, allowing us to adapt to evolving cloud architectures and technologies without being constrained by proprietary solutions.
+* **Broad Tooling Compatibility**: Leveraging CAPI allows us to leverage a multitude of tools developed by the community, enhancing our ability to automate, manage, and orchestrate Kubernetes clusters effectively.
+* **Support for Managed Control Planes**: CAPI enables us to seamlessly integrate with managed control planes like EKS and AKS, providing customers with a choice of deployment options.
+* **Simplified Provider Addition**: CAPI's composable architecture makes it easier to add new infrastructure providers, expanding our offerings beyond AWS, Azure, and our current on-premises solutions.
+* **Multi-Cloud Capabilities**: CAPI supports managing workload clusters from multiple providers under a single management cluster, enabling true multi-cloud deployment scenarios.
+
+## Differences between CAPI and Legacy Cluster Management
+
+The transition to Kubernetes Cluster API (CAPI) introduces several noteworthy differences compared to our legacy cluster management infrastructure. These changes aim to enhance flexibility, expand tooling compatibility, and streamline cluster management operations.
+
+### Cluster CRs Packaged as Helm Charts
+
+CAPI-based cluster configuration is now encapsulated within Helm charts, providing a more controlled and maintainable approach. This approach offers several benefits:
+
+* **Limited Configuration Exposure:** Only tested and stable configuration options are exposed, ensuring a consistent and reliable cluster experience.
+
+* **Ease of Configuration Enabling:** Most CAPI features can be easily enabled by updating the Helm chart, enabling rapid adoption of new capabilities.
+
+* **Inspectable Cluster Assembly:** The Helm chart details clearly outline how the cluster is assembled, providing transparency and diagnostic capabilities.
+
+* **Documented Configuration Options:** Comprehensive documentation is provided as part of the JSON schema, ensuring easy understanding and reference.
+
+* **Happa UI-Based Cluster Creation:** The Happa UI streamlines cluster creation from the JSON schema, enhancing user experience.
+
+### Configurable Registries with Credentials and Mirrors
+
+CAPI enables the flexibility to specify custom registries with credentials and mirrors, allowing customers to utilize their preferred container repositories. This flexibility enhances the portability and adaptability of customer environments.
+
+### Removal of Cluster Management Operators
+
+The legacy cluster management operators have been removed due to several limitations and drawbacks:
+
+* **Incompatibility with Multi-Version CAPI:** Upstream CAPI doesn't support running multiple versions of operators on the same management cluster, creating operational challenges.
```

Comment:
> creating operational challenges.

I don't see this as a limitation TBH, but just a different way to approach it. Also, it seems the CAPI architecture is not ideal.

Reply:
Yes this is not true. It's the other way around.